### PR TITLE
Added clarification to transfer instructions

### PR DESCRIPTION
**Pull request by marciocloudflare**
While we work on revamping instructions, added a bit of clarification to transfer instructions

### CODE REVIEW

File: products/registrar/src/content/transfer-instructions/enom.md [hunk 1]
```diff
@@ -50,10 +50,17 @@ If you do not receive the code, please check the registrant email address listed
 
 --------
 
-## Step 4: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 4: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/enom.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

File: products/registrar/src/content/transfer-instructions/enom.md [hunk 2]
```diff
@@ -50,10 +50,17 @@ If you do not receive the code, please check the registrant email address listed
...
+## Step 5: Transfer to Cloudflare
+After your previous registrar approves the transfer, go to your Cloudflare dashboard. From your accounts home, click **Registrar** > **Transfer** and input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+
+If you do not see this screen or do not have an authorization code, please ensure you have gone through all the steps mentioned above before trying to transfer your domain to Cloudflare.
 
 --------
 
-## Step 5: Approve the transfer
+## Step 6: Approve the transfer
 Once Cloudflare processes your transfer, Enom will send an email to the domain’s registrant contact (the same address where the authorization code was emailed). The email will contain a link that you can click to immediately approve and process the transfer. If you do not click the link, the transfer will automatically approve within 5 days.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/enom.md]: "be approved"?

File: products/registrar/src/content/transfer-instructions/generic.md [hunk 3]
```diff
@@ -50,10 +50,17 @@ Next, your new registrar needs to confirm with your old registrar that the trans
 
 --------
 
-## Step 4: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 4: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/generic.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

File: products/registrar/src/content/transfer-instructions/generic.md [hunk 4]
```diff
@@ -50,10 +50,17 @@ Next, your new registrar needs to confirm with your old registrar that the trans
...
+## Step 5: Transfer to Cloudflare
+After your previous registrar approves the transfer, go to your Cloudflare dashboard. From your accounts home, click **Registrar** > **Transfer** and input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+
+If you do not see this screen or do not have an authorization code, please ensure you have gone through all the steps mentioned above before trying to transfer your domain to Cloudflare.
 
 --------
 
-## Step 5: Approve the transfer
+## Step 6: Approve the transfer
 Your previous registrar will email you to confirm your request to transfer away. Most registrars will include a link to confirm the transfer request. If you click that link, you can accelerate the transfer operation. If you do not act on the email, the registrar can wait up to five days to process the transfer to Cloudflare.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/generic.md]: should this step be worded in the same way Step 5 is worded in Enom?

File: products/registrar/src/content/transfer-instructions/godaddy.md [hunk 5]
```diff
@@ -52,10 +52,17 @@ It may take up to 24 hours for GoDaddy to email you the code. If you do not rece
 
 --------
 
-## Step 4: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 4: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/godaddy.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

File: products/registrar/src/content/transfer-instructions/oneandone.md [hunk 6]
```diff
@@ -48,10 +48,17 @@ Below the “Domain transfer lock” section will be a row listed as “Change p
 
 --------
 
-## Step 4: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 4: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/oneandone.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

File: products/registrar/src/content/transfer-instructions/network-solutions.md [hunk 7]
```diff
@@ -49,10 +49,17 @@ If you do not receive the code, please check the registrant email address listed
 
 --------
 
-## Step 3: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 3: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/network-solutions.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

File: products/registrar/src/content/transfer-instructions/namecheap.md [hunk 8]
```diff
@@ -50,10 +50,17 @@ While still on the **Sharing & Transfer** tab, click the button “Auth Code”
 
 --------
 
-## Step 4: Transfer to Cloudflare
-In the Cloudflare transfer screen, input the authorization code. Each domain will have a unique authorization code and you will need to enter each for every domain you want to transfer.
+## Step 4: Add domain to Cloudflare
+Login to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```
Comment (abracchi-tw) [inline products/registrar/src/content/transfer-instructions/namecheap.md]:
```suggestion
Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login). Choose the account where you want to add the domain. From the accounts home, click **Add a Site** to add the new domain to your Cloudflare account.
```

**Review by abracchi-tw — APPROVED** [review]

File: products/registrar/src/content/transfer-instructions/enom.md (hunk 2 above)
Comment (marciocloudflare) [reply products/registrar/src/content/transfer-instructions/enom.md]: Yes. This is one of those things I've corrected in the Gdoc for the revamp. Good catch :)

File: products/registrar/src/content/transfer-instructions/generic.md (hunk 4 above)
Comment (marciocloudflare) [reply products/registrar/src/content/transfer-instructions/generic.md]: Yup

**Pull request merged**